app/assets/javascripts/index/element.js

```diff
@@ -101,4 +110,83 @@
       scrollableList.scrollLeft = scrollableList.scrollWidth - scrollableList.offsetWidth;
     }
   }
+
+  function previewWikidataValue($btn) {
+    if (!OSM.WIKIDATA_API_URL) return;
+    const items = $btn.data("qids");
+    if (!items?.length) return;
+    $btn.addClass("disabled").removeClass("wdt-preview");
+    fetch(OSM.WIKIDATA_API_URL + "?" + new URLSearchParams({
+      action: "wbgetentities",
+      format: "json",
+      origin: "*",
+      ids: items.join("|"),
+      props: "labels|sitelinks|claims|descriptions",
+      languages: langs.join("|"),
+      sitefilter: wikis.join("|")
+    }), { signal: abortController?.signal })
+      .then(response => response.ok ? response.json() : Promise.reject(response))
+      .then(({ entities }) => {
+        if (!entities) return Promise.reject(entities);
+        $btn
+          .closest("tr")
+          .after(
+            items
+              .filter(qid => entities[qid])
+              .map(qid => getLocalizedResponse(entities[qid]))
+              .filter(data => data.label || data.icon || data.description || data.article)
+              .map(data => renderWikidataResponse(data, $btn))
+          );
+      })
+      .catch(() => $btn.removeClass("disabled").addClass("wdt-preview"));
+  }
+
+  function getLocalizedResponse(entity) {
+    const localizedProperty = (property, langs) => langs.reduce((out, lang) => out ?? entity[property][lang], null);
+    const data = {
+      qid: entity.id,
+      label: localizedProperty("labels", langs)?.value,
+      icon: ["P8972", "P154", "P14"].reduce((out, prop) => out ?? entity.claims[prop]?.[0]?.mainsnak?.datavalue?.value, null),
```

Comment:
For reference:

* [small logo or icon <small>(P8972)</small>](https://www.wikidata.org/wiki/Property:P8972)
* [logo image <small>(P154)</small>](https://www.wikidata.org/wiki/Property:P154)
* [traffic sign <small>(P14)</small>](https://www.wikidata.org/wiki/Property:P14)

P14 surprised me at first, but then I remembered that this would probably result in displaying a route marker alongside a `wikidata=*` tag when viewing a route relation. That does make some sense to me.